tests/flashtest.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Library         SSHLibrary
+
+*** Test Cases ***
+Open Connection and Log In
+    Open Connection  %{OPI_IP}    timeout=360 seconds
+    Login  root    armbian1234
+    
+Flash firmware
+	# turn APU on and put it in S5 state
+	Write    ./relays.sh ON\n
+	# wait for diodes to turn off
```

Comment:
@miczyg1 - @pietrushnic meant leaving some info about checking the led state to track this possibility when we will need it...